docker-compose.override.dev.yml

```diff
@@ -9,6 +9,7 @@ services:
       DD_ADMIN_USER: "${DD_ADMIN_USER:-admin}"
       DD_ADMIN_PASSWORD: "${DD_ADMIN_PASSWORD:-admin}"
       DD_EMAIL_URL: "smtp://mailhog:1025"
+      DD_API_TOKEN_AUTH_ENDPOINT_ENABLED: "False"
```

Comment:
Is there a specific reason to disable this in the `dev` override config and not others? Seems like a development environment would be pretty low-risk.

Reply:
I'm sorry. It is leftover from testing. I will remove it.

Reply:
I'm sorry. It is leftover from testing. I will remove it.